ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -1306,6 +1306,14 @@ class JITModBuilder {
               continue;
             }
 
+            // Vararg related intrinsics appear to be always inlined.
+            if (IID == Intrinsic::vastart || IID == Intrinsic::vaend)
+              continue;
+
+            // Other intrinsics that appear to be always inlined.
+            if (IID == Intrinsic::smax || IID == Intrinsic::usub_with_overflow)
```

Comment:
Obvious question: is `smin` inlined? Basically is there some way we can interrogate the intrinsic to see if it will be inlined?

Reply:
It probably is, along with some others. But I didn't want to add them without actually seeing them in a test.

Reply:
I'll ask about:

>  is there some way we can interrogate the intrinsic to see if it will be inlined?

Reply:
It's really hard. We looked into this a few weeks ago and gave up. There are a kajillion places in SelectionDAGIsel which could inline (or not) an intrinsic and we'd have to find and instrument them all.

It's on our wishlist to find a solution to this.

Reply:
@ptersilie did we decide that `-O0` only ever uses `FastISel`?

Reply:
I forget exactly how it works now, but I think Lukas could reliably label intrinsic inlinings in FastISel but not SelectionDAG.

Reply:
OK, that sounds like we should file an issue and do that in a future (not too future!) PR?